cadc-permissions/src/main/java/org/opencadc/permissions/TokenTool.java

```diff
@@ -238,19 +242,21 @@ public String validateToken(String token, URI expectedURI, Class<? extends Grant
                 user = value;
             }
         }
-        log.debug("uri: " + uri);
-        log.debug("grant: " + grant);
-        log.debug("subject: " + user);
-        
+        log.debug("[TokenTool.validateToken]: uri: " + uri);
```

Comment:
Having to put the TokenTool class name here is a sign of our generally poor logging config, where we only log the message (info level start and end messages are json) so we lose out on all the stuff log4j does for us :-(

and to top it off, the json makes logs unreadable and we don't ingest them into something that can search anyway :-( :-(

Reply:
This could have also been fixed more widely by fixing the Log4jInit to use a proper log format for debug messages